images/chromium-headful/client/src/plugins/globalPaste.ts

```diff
@@ -0,0 +1,30 @@
+// src/plugins/globalPaste.ts
+import { PluginObject } from 'vue'
+
+const GlobalPaste: PluginObject<undefined> = {
+  install() {
+    document.addEventListener(
+      'keydown',
+      async (e: KeyboardEvent) => {
+        if ((e.ctrlKey || e.metaKey) && e.key.toLowerCase() === 'v') {
+          e.preventDefault()
+          console.log(`[vue:globalPaste]:call`)
+          try {
+            const text = await navigator.clipboard.readText()
+            console.log(`[vue:globalPaste]:payload:` , text)
+            await fetch('http://localhost:10001/computer/paste', {
```

Comment:
using localhost here won't work in prod

Reply:
yes definitely--this is why we added the click/move mouse stuff into the API. I think eventually we will go deeper on that front